index.html

```diff
@@ -1305,37 +1305,49 @@ <h1>Input Range</h1>
         <!-- Grid Generator -->
         <div data-content="grid-generators" class="grid-generators">
           <label for="grid-generators-preview">
-            <p>CSS Grid Preview</p>
-            <div
-              id="grid-generators-preview"
-              class="grid-generators-preview"
-            ></div>
+            <h1>CSS Grid Preview</h1>
```

Comment:
this part shouldn't be inside the label, you can see how it's done in other generators

Reply:
done @Dun-sin